### PR TITLE
fix: Add more space between buttons and event card (#9119)

### LINKED ISSUE #9119: [BUG] Add more space between the buttons and the event card, when adding and editing the event(Mobile view).
### Description

In mobile view when I am trying to add and edit an event there's no proper spacing between the buttons and the event card.

### Screenshots

when adding an event, there's no space between the save button and the event card:

add-event

when editing the card, there's no space between the buttons(save / delete) and the event card :

edit-event

 

### Do you want to work on this issue?

Yes

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ezhil56x**
## Fixes Issue
Closes #9119

## Changes proposed

I have added marginBottom 1.5 rem through inline style element

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

image

### CODE REVIEW

**Comment by AbhishekBhure** [comment]
looks good!!

File: pages/account/manage/event/[[...data]].js [hunk 1]
```diff
@@ -336,11 +336,11 @@ export default function ManageEvent({ BASE_URL, event }) {
 
                 <div className="mt-6 flex items-center justify-end gap-x-6">
                   {event._id && (
-                    <Button type="button" onClick={() => setOpen(true)}>
+                    <Button type="button" onClick={() => setOpen(true)} style={{marginBottom: '1.5rem'}}>
```
Comment (Dun-sin) [inline pages/account/manage/event/[[...data]].js:339]: as someone already said, you can do this in tailwind unless you are calculating a value, you can just commit my change, should be faster for you, your choice
```suggestion
                    <Button type="button" onClick={() => setOpen(true)} className='mb-6'>
```

File: pages/account/manage/event/[[...data]].js [hunk 2]
```diff
@@ -336,11 +336,11 @@ export default function ManageEvent({ BASE_URL, event }) {
...
                       DELETE
                     </Button>
                   )}
-                  <Button type="submit" primary={true}>
+                  <Button type="submit" primary={true} style={{marginBottom: '1.5rem'}}>
```
Comment (Dun-sin) [inline pages/account/manage/event/[[...data]].js:343]:
```suggestion
                  <Button type="submit" primary={true} className='mb-6'>
```

**Review by Dun-sin — CHANGES_REQUESTED** [review]
This change should make your PR perfect

File: pages/account/manage/event/[[...data]].js (hunk 2 above)
Comment (ezhil56x) [reply pages/account/manage/event/[[...data]].js:343]: I tried adding mb-6 to the classname as you have suggested but that didn't work, so I used style element

**Using className='mb-6'**

image

image

Comment (Dun-sin) [reply pages/account/manage/event/[[...data]].js:343]: let me get this straight, my change causes the save button to also lose its styling of colour etc

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please use Tailwind classes

**Comment by ezhil56x** [comment]
> Thank you for your contribution. Please use Tailwind classes

Hi, I have changed to Tailwind classes as suggested. Is there anything more I need to change?  Happy to do 😄

File: pages/account/manage/event/[[...data]].js (hunk 2 above)
Comment (ezhil56x) [reply pages/account/manage/event/[[...data]].js]: I have successfully implemented by using Tailwind classes

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**